bsp/esp32_p4_function_ev_board/idf_component.yml

```
@@ -0,0 +1,18 @@
+version: "1.0.0"
+description: Board Support Package for ESP32-P4 Function EV Board
+url: https://github.com/espressif/esp-bsp/tree/master/bsp/esp32_p4_function_ev_board
+
+targets:
+  - esp32p4
+
+dependencies:
+  idf: ">=5.0"
```

P4 preview support started in 5.2
```suggestion
  idf: ">=5.2"
```